src/bin/travis/testCentaurCwlConformance.sh

```diff
@@ -19,8 +20,12 @@ shutdownCromwell() {
 
 trap "shutdownCromwell" EXIT
 
+# Start the Cromwell server in the directory containing input files so it can access them via their relative path
+CURRENT_DIR=$(pwd)
+cd CWL_TEST_DIR
 java -jar ${CROMWELL_JAR} server &
 CROMWELL_PID=$$
+cd CURRENT_DIR
```

Comment:
you might subshell this instead

Reply:
If I subshell it will `CROMWELL_PID` still be defined outside the subshell ?

Reply:
ugh nm. you can do disgusting things with globals but that's totally not worth it.